gtest/src/manager.rs

```diff
@@ -402,13 +393,58 @@ impl ExtManager {
 
     /// Call non-void meta function from actor stored in manager.
     /// Warning! This is a static call that doesn't change actors pages data.
-    pub(crate) fn call_meta(
+    pub(crate) fn read_state(&mut self, program_id: &ProgramId) -> Result<Vec<u8>> {
```

Comment:
Isn't it better to decode result (or provide alternative functions with decoded output)?